README.md

```diff
@@ -90,6 +90,8 @@ let g:minimap_auto_start_win_enter = 1
 | `g:minimap_highlight_range`                   | `0`                                                       | if set, minimap will highlight range of visible lines                |
 | `g:minimap_highlight_search`                  | `0`                                                       | if set, minimap will highlight searched patterns                     |
 | `g:minimap_git_colors`                        | `0`                                                       | if set, minimap will highlight range of changes as reported by git   |
+| `g:minimap_set_highlights`                    | `1`                                                       | if set, minimap will create an autocommand to set highlights on color scheme changes. |
```

Comment:
I think this name would benefit from being a bit more verbose. Some thing like `minimap_enable_highlight_colorgroup`.